specs/sequencer-app.md

```diff
@@ -41,143 +42,309 @@ idea of "messages", where every transaction can contain multiple messages. The
+## [Actions](https://buf.build/astria/protocol-apis/docs/main:astria.protocol.transaction.v1)
+
+The following is an exhaustive list of all user-accessible actions available
+to be submitted as of Sequencer v1.0.0-rc.1.
+
+### Core Protocol Actions
+
+* `Transfer`: represents a value transfer action between two accounts. It consists
+of the following fields:
+
+  | **Field** | **Type** | **Description** |
+  | --------- | -------- | ----------- |
+  | to        | `Address`| The account to transfer to. The "from" address is assumed|
+  ||| to be the signer of the transaction. |
+  | amount    | `uint128`| The amount to transfer. |
+  | asset     | `string` | The asset to transfer. |
+  | fee_asset | `string` | The asset used to pay for the action's fees. |
+
+* `RollupDataSubmission`: a transaction ordered by the sequencer, whose destination
+is another chain. It consists of the following:
+
+  | **Field** | **Type** | **Description** |
+  | --------- | -------- | ----------- |
+  | rollup_id | `RollupId`| ID of the destination chain. |
+  | data      | `bytes`  | The opaque transaction data. |
+  | fee_asset | `string` | The asset used to pay for the action's fees. |
+
+### Bridge Actions
+
+These actions deal with transfering funds to/from a bridge account to be used on
+a rollup.
+
+* `InitBridgeAccount`: initializes a bridge account for the given rollup on the
+sequencer chain. The `withdrawer_address` is the only actor authorized to transfer
+out of the account. It is set by default to the signer of the transaction, but
+can also be set to a different address within the action itself, or changed later
+via a `BridgeSudoChange` action. `InitBridgeAccount` consists of the following
+fields:
+
+  | **Field** | **Type** | **Description** |
+  | --------- | -------- | ----------- |
+  | rollup_id | `RollupId` | The rollup ID with which to register the bridge account.|
+  | asset     | `string`   | The asset ID that will be accepted by the account.|
+  | fee_asset | `string`   | The asset with which to pay fees. |
+  | sudo_address | `Address` | The address which has authority over the bridge|
+  ||| account. If empty, assigned to the signer. |
+  | withdrawer_address | `Address` | The address which is allowed to withdraw funds|
+  ||| from the bridge account. If empty, assigned to the signer.|
+
+* `BridgeLock`: transfers funds from a sequencer account to a bridge account.
+It is effectively similar to `Transfer`. Upon execution of a bridge lock action,
+a `Deposit` event will be included in the block data for the rollup this bridge
+account is registered to, containing the information of the transfer.
+
+  | **Field** | **Type** | **Description** |
+  | --------- | -------- | ----------- |
+  | to        | `Address`| The bridge account to transfer to. The "from" address|
+  ||| is assumed to be the signer of the transaction. |
+  | amount    | `uint128`| The amount to transfer. |
+  | asset     | `string` | The asset to transfer. |
```

Comment:
unrelated to this doc, but this seems like an unnecessary parameter since bridge accounts only have one asset. can make an issue